ale/base/type_sensor.py

```diff
@@ -362,3 +368,125 @@ def line_times(self):
         : array
         """
         raise NotImplementedError
+
+
+class Cahvor():
```

Comment:
We should also add a property for sensor_position. This should be the C vector + the vector to the center of the body and then all of that rotated into J2000.